grammars/javascript.cson

```diff
@@ -41,7 +41,7 @@
         'name': 'meta.brace.curly.js'
       }
     ]
-    'end': '(;)?$'
+    'end': '(;)|$'
```

Comment:
Related to my comment below (or above I guess, depending on how you look at it): Can this be made into `(?=;|$)` to prevent the `;` from also getting tokenized as `meta.import.js` and also to remove the `endCaptures` block?